comms/c3270/Portfile

```diff
@@ -26,18 +26,17 @@ depends_lib         path:lib/libssl.dylib:openssl  \
                     port:libiconv
 
 distfiles           suite3270-${version}-src.tgz
-dist_subdir         x3270
-checksums           md5     e8106ab4240d996bd2cea53df14634ae \
-                    sha1    e911b66fc73048b8d99bbc1c27a88b27d6fdd1b6 \
-                    rmd160  64dafc97841c773b5236c2aeebca62882203d089 \
-                    sha256  e22f40360170acf70b6de521173c633072582315b4879aef276fb8c97102e848
+dist_subdir         c3270
+checksums           md5     e71b1501e2abe760fb029e19da409e39 \
+                    sha1    2574fad86813309ab6392c5d18a1febfa24c7616 \
```

Comment:
The `md5` and `sha1` checksums are no longer used - we can remove them and only keep `rmd160`, `sha256`, and `size`.